tests/LanguageServerTest.php

```diff
@@ -36,7 +36,10 @@ public function testInitialize()
                 'textDocumentSync' => TextDocumentSyncKind::FULL,
                 'documentSymbolProvider' => true,
                 'hoverProvider' => true,
-                'completionProvider' => null,
+                'completionProvider' => (object)[
+                    'resolveProvider' => false,
+                    'triggerCharacters' => ['$', '>']
```

Comment:
That is because JSON does not have a "character" type, only a string type. But the property name clearly says trigger_Characters_, so if trigger _sequences_ are supported, it should be clarified in the protocol documentation.